solr/solr-ref-guide/modules/getting-started/pages/tutorial-paramsets.adoc

```diff
@@ -0,0 +1,147 @@
+= Exercise 4: Using ParamSets
+:experimental:
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+[[exercise-4]]
+== Exercise 4: Using ParamSets
+
+This exercise will teach you to use ParamSets to group a number of different query parameters into a labelled grouping that you can refer to in your queries.
+
+=== Getting Ready
+
+Make sure you have a running Solr, following the steps in xref:tutorial-films.adoc#restart-solr[].
+Then go ahead to the next section.
+
+=== Create a New Collection
+
+[,console]
+----
+$ bin/solr create -c films
+----
+
+Because we didn't specify a ConfigSet, we will end up using the `_default` ConfigSet.
+We'll specify the specific schema for a couple of fields that Solr would otherwise guess differently (than we'd like) about:
+
+[,console]
+----
+$ curl http://localhost:8983/solr/films/schema -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field" : [
+    {
+      "name":"name",
+      "type":"text_general",
+      "multiValued":false,
+      "stored":true
+    },
+    {
+      "name":"initial_release_date",
+      "type":"pdate",
+      "stored":true
+    }
+  ]
+}'
+----
+
+[NOTE]
+====
+Without explicitly defining those field types, the _name_ field would have been guessed as a multi-valued string field type
+and _initial_release_date_ would have been guessed as a multi-valued `pdate` type.  It makes more sense with this
+particular data set domain to have the movie name be a single valued general full-text searchable field,
+and for the release date also to be single valued.
+====
+
+=== Index the Data
+
+Now that we have updated our Schema, we need to index the sample film data, or, if you already have indexed it, then re-index it to take advantage of the new field definitions we added.  
+Look back at the xref:tutorial-films.adoc#index-sample-film-data[] for the specific steps.
```

Comment:
Exercise 4 here says to look back but Exercise 5 repeats the steps. Perhaps Exercise 5 could say to look back too for consistency?

Reply:
I am actually thinking we should just embed the command.  it's not a lot...  and help someone not jump back and forth...

Reply:
I may go back and tweak the startup commands and copy them too...